examples/parquet_analytics_demo.ipynb

```diff
@@ -0,0 +1,470 @@
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4638e141",
+   "metadata": {},
+   "source": [
+    "Next, let's make a toy assumption for the purpose of demonstration. Let's create a \"fake\" status for each patient in our data by treating the last digit of their phone number as if it was indicative of some sort of status or test (this is obviously completely random, but it helps bear out the graph). We'll begin by extracting this last digit into its own \"status\" field, and then we'll visualize a histogram of the results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "668c979f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Extract last digit of phone number into its own field and convert to numeric\n",
+    "patients_with_phones <- patients[!(is.na(patients$phone_number) | patients$phone_number == \"\"),]\n",
+    "patients_w_status <- patients_with_phones %>%\n",
+    "  mutate(\n",
+    "    status=as.numeric(stringr::str_sub(phone_number,-1,-1))\n",
+    "  )\n",
+    "\n",
+    "# Now let's visualize what that looks like\n",
+    "ggplot(patients_w_status) +\n",
+    "    geom_histogram(aes(x=status),fill=\"green\",color=\"white\",bins=10) +\n",
+    "    ggtitle(\"Patient Status Test\") + theme_minimal()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "be022140",
+   "metadata": {},
+   "source": [
+    "### A Choropleth Visualization Of Patients"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4046bf3a",
+   "metadata": {},
+   "source": [
+    "For our final example, let's do something a little more applicable. Using the joined data we created in the first example above, let's create a choropleth visualization of this data disaggregated across the state of Virginia (a _choropleth_ chart is a map where each region is color-coded by intensity of some numerical variable, like a heatmap). We'll be using the most 2018 tracts information on counties and subcounties published by Virginia itself, available [here](https://catalog.data.gov/dataset/tiger-line-shapefile-2018-state-virginia-current-census-tract-state-based). While it's possible to use data formats such as GeoJSON for spatial analysis, we'll opt for a shapefile, as that's the preferred layer format used by most US Federal agencies and surveys (such as the Census). We'll be performing only a basic analysis, creating a choropleth for all patients in our dataset with preexisting conditions, but the points made below are applicable to a wide variety of geospatial applications.\n",
+    "\n",
+    "Let's start by reading in our geospatial data for the state of Virginia and visualizing what the map looks like when blank."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2b76e6e2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Open the shapefile using the sf package\n",
+    "va_tracts = st_read(\"/Users/Brandon/Downloads/tl_2018_51_tract/tl_2018_51_tract.shp\")\n",
```

Comment:
I think we should replace this location with a local variable and a dummy location